glutin/src/prelude.rs

```diff
@@ -0,0 +1,17 @@
+//! The glutin prelude.
+//!
+//! The purpose of this module is to bring common imports, given that all
+//! graphics api are on the traits for the documetation sharing purposes.
```

Comment:
Attempted rewording using the unix prelude from Rust for reference.

```suggestion
//! The purpose of this module is to make accessing common imports more convenient.
//! The prelude also imports traits shared by the implementations of graphics apis.
```